app/setup.py

```diff
@@ -64,8 +64,12 @@ def create_app(setting_overrides=None):  # noqa: C901  pylint: disable=too-compl
 
     application.eq = {}
 
-    secrets = yaml.safe_load(open(application.config['EQ_SECRETS_FILE']))
-    keys = yaml.safe_load(open(application.config['EQ_KEYS_FILE']))
+    with open(application.config['EQ_SECRETS_FILE']) as secrets_file:
+        secrets = yaml.safe_load(secrets_file)
+
+    with open(application.config['EQ_KEYS_FILE']) as keys_file:
+        keys = yaml.safe_load(keys_file)
```

Comment:
@jonnyshaw89 This may be a silly question but how does this ensure they get closed ?

Reply:
Have a google

Reply:
🥇